test/test_quantized_tensor.py

```diff
@@ -3,10 +3,19 @@
 import numpy as np
 
 import torch
+import pickle
 
 from common_utils import TestCase, run_tests
 import tempfile
 
+class Foo(torch.nn.Module):
+    def __init__(self):
+        super(Foo, self).__init__()
+        self.dtype = torch.per_tensor_symmetric
```

Comment:
per_tensor_symmetric should be a qscheme, not a dtype.  x.to(torch.per_tensor_symmetric) should be an error.
We should not be able to do x.to(torch.qint8), even though qint8 is a valid dtype as we do not have a scale and zero-point.

Reply:
Oh this is just left over from me copy-pasting everything about dtype